test/langtools/tools/javac/LocalInterface.java

```diff
@@ -2,8 +2,8 @@
  * @test  /nodynamiccopyright/
  * @bug 8242478
  * @summary test for local interfaces
- * @compile/fail/ref=LocalInterface.out -XDrawDiagnostics LocalInterface.java
- * @compile --enable-preview -source ${jdk.version} LocalInterface.java
```

Comment:
the current patch is supposed to be applied after the code being reviewed at [PR-290](https://github.com/openjdk/jdk/pull/290) which also modifies this particular line, once [PR-290](https://github.com/openjdk/jdk/pull/290) has been pushed this line and similar should change